#!/bin/sh -e
## 01_aserver_options.dpatch by Thomas Hood <jdthood@yahoo.co.uk>
##
## DP: Terminate long options list

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/jdthood/src/pkg-alsa/pkg-alsa/trunk/build-area/alsa-lib-1.0.6/aserver/aserver.c alsa-lib-1.0.6/aserver/aserver.c
--- /home/jdthood/src/pkg-alsa/pkg-alsa/trunk/build-area/alsa-lib-1.0.6/aserver/aserver.c	2004-09-23 21:05:08.000000000 +0200
+++ alsa-lib-1.0.6/aserver/aserver.c	2004-09-23 21:06:10.000000000 +0200
@@ -1006,7 +1006,8 @@
 int main(int argc, char **argv)
 {
 	static struct option long_options[] = {
-		{"help", 0, 0, 'h'}
+		{"help", 0, 0, 'h'},
+		{ 0, 0, 0, 0 }
 	};
 	int c;
 	snd_config_t *conf;
